Makefile

```diff
@@ -42,7 +42,7 @@ test-gentype:
 test-all: test test-gentype
 
 reanalyze:
-	reanalyze.exe -set-exit-code -all-cmt _build/default/res_syntax -suppress res_syntax/testrunner
```

Comment:
Does this mean res_syntax is not checked anymore?

Reply:
It's an exclusion list now

Reply:
But it only checks _build/default/jscomp, and res_syntax is not in jscomp. Or am I missing something?

Reply:
Oh right I haven't updated the path.